src/ui/Pricing/SubscriptionSettings/SubscriptionCard/PlanCard.svelte

```diff
@@ -69,7 +69,7 @@
   badgeIcon={discount ? null : badgeIcon}
 >
   <slot>
-    <p>
+    <p class="body-2">
```

Comment:
`body-2` class breaks existing desktop UI. Please, make sure it's consistent with Figma design